api/types/constants.go

```diff
@@ -353,6 +353,10 @@ const (
 	// imported from a cloud provider.
 	OriginCloud = "cloud"
 
+	// OriginCloudDiscovery is an origin value indicating that the resource
+	// was imported from a cloud provider through discovery service.
+	OriginCloudDiscovery = "cloud-discovery"
```

Comment:
Kubernetes & Server discovery uses `OriginCloud` as the origin value.
We should stick with one instead of having different values for each resource type

Reply:
It could hard to differentiate if the resource is discovered by the db service or discovery service (especially at the db service). 

Is the plan to kill discovery in the db service eventually?